docs/general/dns-providers.md

```diff
@@ -1125,3 +1125,19 @@ Blocks more ads and malware than AdGuard DNS thanks to more advanced syntax, but
 | DNS, IPv4      | Varies; see link above. |
 | DNS, IPv6      | Varies; see link above. |
 | DNSCrypt, IPv4 | Varies; see link above. |
+
+#### dnslow.me
+
+[dnslow.me](https://dnslow.me/) is an advertisement and threat blocking, privacy-first, encrypted DNS, also a free and open source project on GitHub.
```

Comment:
I suggest dividing this sentence in multiple parts.

`[dnslow.me](https://dnslow.me/) is a free privacy-first DNS service that blocks ads and protects you from threats. Its source code is available on GitHub.`